### ISSUE TITLE
Frogatto 4 needs a trailer.

### ISSUE THREAD

**Issue by DDR0** [frogatto/frogatto#625]
Related to #624, we need a nice trailer for our Steam page. It would be useful for announcements and such on our blog, as well.

**Comment by al0fdf** [comment]
I believe [the trailer](https://video.fastly.steamstatic.com/store_trailers/256951788/movie480.mp4?t=1686370301) is on the [steam page](https://store.steampowered.com/app/865030/Frogatto__Friends/) now, and the issue can be closed:

**Comment by DDR0** [comment]
Yeah, we can definitely close this. Thanks for flagging it... twice...
